internal/events/event_poller.go

```diff
@@ -217,7 +217,9 @@ func (ep *eventPoller) eventLoop() {
 		}
 
 		// Once we run out of events, wait to be woken
-		if !repoll {
+		if repoll {
+			time.Sleep(10 * time.Millisecond)
```

Comment:
Without sleeping, if using `repoll` its a hot spin. Is that intended? If there should be some sort of repoll delay, I'm assuming that should be configurable?

Reply:
I'd like to understand this one a bit more. `repoll` is intended to be returned when the pluggable function knows there's more to do, so it shouldn't be a hot spin. But if you've found an implementation of `newEventsHandler` that has a hot spin condition would love to talk it through.

I believe the most common case to want to immediately read again, is if you've read a full page - so there's more potentially there that simply were beyond your page boundary.

Reply:
> But if you've found an implementation of newEventsHandler that has a hot spin condition would love to talk it through.

Only one that I created bc I was mis-using `repoll` 😄 . That makes sense, thanks for explaining that. Will remove this.